src/react/components/_dashboard/NotificationsCard/ArchiveLink.js

```diff
@@ -6,6 +6,7 @@ import APILink from 'react/components/APILink';
 const ArchiveLink = ({ url }) => {
```

Comment:
I see the URL being used here being sourced from `CampusSolutions::DashboardUrl.new.get`

We should refactor this to use CS Config and put it in the Link API (not today though).

That will allow us to get away from explictly having to define `showNewWindow: false` on line 9

Reply:
Agreed, I discussed this with Barbara also